### PR TITLE
Addressbook For Actor Identification

### PR DESCRIPTION
**Pull request by yozohu**
* re-introduced address book, for logging identification

### CODE REVIEW

File: bellboy/actors/generic.py [hunk 1]
```diff
@@ -15,9 +15,26 @@ def __init__(self, *args, **kwargs):
...
+
+    def _nameAddress(self, address: ActorAddress, name: str):
+        """
+        Adds entry to my address book.
+        """
+        if name is not None:
+            # using the str of the address as key bc the address itself is unhashable type
+            self._address_book[str(address)] = name
+
     def createActor(
```
Comment (RyanFleck) [inline bellboy/actors/generic.py:38]: Is this overridden/called when an actor is created @yozohu ?

**Review by RyanFleck — APPROVED** [review]
_LGTM_

File: bellboy/actors/generic.py (hunk 1 above)
Comment (yozohu) [reply bellboy/actors/generic.py:38]: Yeah, this createActor is a wrapper for thespians actor.createActor (not actorSystem.createActor tho).  
It creates the actor, and configures its logging to fit our system, and adds its global name to relevant addressbooks.

Comment (yozohu) [reply bellboy/actors/generic.py:38]: i should add a comment in the code, mb

**Pull request merged**